docs: Nautilus dev tools page (#25928)
MystenLabs/sui: docs/content/guides/developer/nautilus/community-dev-tools.mdx

:::

## Deploy Dockerized Nautilus apps to Marlin

Marlin Oyster integrates with the Nautilus framework to remove the operational overhead of managing enclave infrastructure.

To use Nautilus, you can either manage AWS Nitro Enclaves yourself or use Marlin Oyster. When you manage enclaves yourself, you must manage AWS accounts, networking configurations, enclave images, and attestation infrastructure.

Marlin Oyster handles deterministic enclave builds, provisioning, and attestation. You deploy enclave-powered apps with only a Docker image, Oyster CLI, and Sui. No direct AWS interaction is required. Marlin Oyster preserves all cryptographic guarantees, including hardware-backed Nitro security, reproducible enclave measurements (Platform Configuration Registers (PCRs)), and trust-minimized execution.

It transforms Trusted Execution Environment (TEE) access into a transparent and decentralized marketplace. A Sui Move contract enables you to submit jobs, pay with stablecoins, and expose workloads to Oyster operators. Builds are deterministic, meaning enclave integrity and identity are fully verifiable.

> You can learn more about Marlin Oyster and Nautilus on the [Marlin blog](https://blog.marlin.org/scaling-confidential-compute-on-sui-nautilus-and-marlin-oyster-integration), or learn more about Marlin in the [Marlin documentation](https://docs.marlin.org/oyster/introduction-to-marlin/).

Reviewer: ```suggestion
You can learn more about Marlin Oyster and Nautilus in [Marlin docs](https://docs.marlin.org/oyster/build-cvm/guides/sui-oyster/).
```
